packages/sommelier.rb

```diff
@@ -360,10 +361,10 @@ def self.build
             # Set default SCALE to 1 if unset.
             SCALE=${SCALE:-1}
             # Allow overriding environment variables before starting sommelier daemon.
-            [ -f "$HOME/.sommelier.env" ] && source ~/.sommelier.env 2>> #{CREW_PREFIX}/var/log/sommelier.log
+            [ -f "#{CREW_PREFIX}/.config/.sommelier.env" ] && source #{CREW_PREFIX}/.config/.sommelier.env 2>> #{CREW_PREFIX}/var/log/sommelier.log
```

Comment:
How about add to `#{CREW_PREFIX}/etc/env.d`?

Reply:
This is for user overrides to the defaults that won't be overwritten by package updates, so that would be inappropriate.